README.md

```diff
@@ -171,6 +172,25 @@ This performs the same initial heating, before running a further 1000 steps (1 p
 When MD is run with heating the trajectory ```NaCl-nvt-T20.0-T300.0-T300.0-traj.xyz``` and statistics ```NaCl-nvt-T20.0-T300.0-T300.0-stats.dat``` files will indicate the heating range and MD temperature (which may be different). With heating and MD trajectories/statistics within the same files.
 
 
+### Equation of State
+
+Fit the equation of state for a structure (using the [MACE-MP](https://github.com/ACEsuit/mace-mp) "small" force-field):
+
+```shell
+janus eos --struct tests/data/NaCl.cif --no-minimize --min-lattice 0.9 --max-lattice 1.1 --n-lattice 9 --arch mace_mp --calc-kwargs "{'model' : 'small'}"
+```
+
+This will save the energies and volumes for nine lattice constants in `NaCl-eos-raw.dat`, and the fitted minimum energy, volume, and bulk modulus in `NaCl-eos-raw.dat`, in addition to generating a log file, `eos.log`, and summary of inputs, `eos_summary.yml`.
```

Comment:
```suggestion
This will save the energies and volumes for nine lattice constants in `NaCl-eos.dat`, and the fitted minimum energy, volume, and bulk modulus in `NaCl-eos.dat`, in addition to generating a log file, `eos.log`, and summary of inputs, `eos_summary.yml`.
```